Formula/apache-arrow.rb

```diff
@@ -22,7 +22,6 @@ class ApacheArrow < Formula
   depends_on "zstd"
 
   def install
-    ENV.cxx11
     args = %W[
       -DARROW_COMPUTE=ON
       -DARROW_CSV=ON
```

Comment:
Yep, I think this is needed. I don't see any other changes in https://github.com/apache/arrow/blame/master/dev/tasks/homebrew-formulae/autobrew/apache-arrow.rb that aren't here. 

```suggestion
      -DARROW_CSV=ON
      -DARROW_CXXFLAGS="-D_LIBCPP_DISABLE_AVAILABILITY"
```